connect/src/com/telenor/connect/ui/ConnectLoginButton.java

```diff
@@ -107,6 +107,11 @@ public void onClick(View v) {
                 parameters.put("scope", TextUtils.join(" ", getLoginScopeTokens()));
             }
 
+            final String mccMncLoginHint = ConnectSdk.getMccMncLoginHint();
```

Comment:
Remember this is an open repo, so Confluence page won't do. But I googled `MCCMNC` and then I understood, so I guess that might be enough for it to make sense for most people.